model/src/main/java/com/basistech/rosette/dm/TranslatedData.java

```diff
@@ -83,18 +116,20 @@ protected MoreObjects.ToStringHelper toStringHelper() {
      * Builder class for TranslatedData.
      */
     public static class Builder extends BaseAttribute.Builder<TranslatedData, TranslatedData.Builder> {
-        private TextDomain domain;
-        private String translation;
+        private final TextDomain sourceDomain;
+        private final TextDomain targetDomain;
+        private final String translation;
         private Double confidence;
 
         /**
          * Constructs a builder from the required properties
          *
-         * @param domain specifies the language and script of the translation
+         * @param targetDomain specifies the language and script of the translation
          * @param translation the translation for the text
          */
-        public Builder(TextDomain domain, String translation) {
```

Comment:
Don’t remove this constructor. Removing a constructor is backwards-incompatible.

Reply:
This also won't show up as outdated but I added this back in line 134. Good catch :slightly_smiling_face: